libs/common/src/components/screens/welcome/layout.tsx

```diff
@@ -150,6 +150,7 @@ export const WelcomeLayout = observer<WelcomeLayoutProps>(
                   ...theme.typography.largeTitle,
                   color: "#F6F5FF",
                   marginTop: theme.spacing["24"],
+                  ...(theme.welcome.buttons.length > 3 && { fontSize: 24 }),
```

Comment:
Is this Evan-approved? IMHO it makes more sense to make the big logo smaller instead of making text smaller and therefore less readable

Reply:
the problem is that the image becomes a dot some times so I made the text readable and the image show, its just that the text is too big and when there are more than 3 buttons the image don't show some times, I showed it to him but he didn't give an answer on that

Reply:
Wdym by "the image becomes a dot some times"?

Changing the font size should be the last resort. This is a hack in the same category as our previous `isSmallScreen`. It goes against accessibility and also makes the font sizes inconsistent with the other screens.

In this specific case, for example, you can scale down the image's height (and the spacing around the image). Furthermore, since the image is purely cosmetic, you can even hide it if there isn't enough space (which in our current version is maybe at around 600px).

Reply:
I mean that some images shrink so much it becomes hard to see so that's why I decided to give it some space by shrinking the text a little 

![photo_2023-07-07 12 36 55](https://github.com/obi-wallet/obi-wallet-internal/assets/2956853/ce9786a0-36e5-4fe8-abf4-3231f3715c23)

![photo_2023-07-07 12 37 00](https://github.com/obi-wallet/obi-wallet-internal/assets/2956853/6b07f856-3cc6-47cf-a634-a9e712b6950a)

that's the difference.

Reply:
I saw the change, still doesn't change my arguments above: don't shrink the text, shrink the image instead (or reduce the spacing around the image).